src/crimson/mgr/client.cc

```diff
@@ -117,7 +117,7 @@ seastar::future<> Client::handle_mgr_conf(ceph::net::Connection* conn,
 
 void Client::report()
 {
-  seastar::with_gate(gate, [this] {
+  (void) seastar::with_gate(gate, [this] {
```

Comment:
could you add a comment to explain why we can or want to ignore the return value of `with_gate()`?

Reply:
added, see commit message of https://github.com/ceph/ceph/pull/30313/commits/16f2d7558264552ccc6868eee0108e180035551d